v4/export/writer_util.go

```diff
@@ -191,12 +191,16 @@ func (s *SQLTypeString) ReportSize() uint64 {
 }
 func (s *SQLTypeString) ToString() string {
 	if s.Valid {
-		return fmt.Sprintf(`'%s'`, s.String)
+		return fmt.Sprintf(`'%s'`, escapeSingleQuote(s.String))
 	} else {
 		return "NULL"
 	}
 }
 
+func escapeSingleQuote(src string) string {
+	return strings.ReplaceAll(src, "'", "\\'")
```

Comment:
The SQL-compliant escaping is `''` not `\'`. 

And if we need to support `\'` we gotta escape `\` as `\\` too.